.travis.yml

```diff
@@ -7,11 +7,12 @@ cache:
 php:
   - '7.1'
   - '7.2'
-  - nightly
+  - '7.3'
+  - '7.4'
 
 env:
   global:
-    - PHPUNIT_FLAGS='--stop-on-failure --verbose'
+    - PHPUNIT_FLAGS='--verbose'
```

Comment:
is there any reason to prevent a fail fast anymore?

Reply:
well, as I explained in the commit message, I find it more useful for a CI to report *all* failing tests than to stop the build fast. It gives you a chance to fix them all before resubmitting rather than having to do it one by one (and it allows maintainers to have an idea about how broken the tests are).

Reply:
I would prefer a fail fast tbh: if tests are broken you are likely going to need to run `make test` to ensure everything works fine and it relieves a bit the CI which is not always the fastest

Reply:
well, I might have green tests locally, because only some PHP versions are broken.

and the complete PHPUnit run takes 1.19s. I don't think a faster failure will change much for the CI.
If you want CI to go faster, a better way would be to avoid the 1min spent in `composer bin all update` (while not all tools are needed by each job, and `covers-validator` could be run in a single job rather than all as well to avoid the need in others). Btw, the failed resolution for `humbug` is probably a culprit there.

Reply:
Yeah I need to review a bit the bundle and bump some dependencies :)